### PR TITLE
redmine and uframe blueprints.  These are where their respective endpoin...

### PR DESCRIPTION
**Pull request by FBRTMaka**
...ts and manipulation files should live

Yay feature branch! :D

### CODE REVIEW

**Comment by FBRTMaka** [comment]
So, to keep things tidy there are two blueprint modules; a uframe and a redmine. The uframe can have stuff like ‘data’ in it and not be ambigous, which is nice.  Also, these routes have a prefix to them.  So, the “/get_data” route must now be preceeded by ‘/uframe’…so ‘/uframe/get_data’.  In the ui world, the complete path would (I suppose) be ‘/api/uframe/get_data’.

**Comment by birdage** [comment]
@FBRTMaka did you rebase on top of the current master?

**Comment by FBRTMaka** [comment]
no way, not till that is working

File: ooiservices/app/main/routes.py [hunk 1]
```diff
@@ -80,47 +78,3 @@ def get_parameters():
 def get_parameter(id):
     parameter = StreamParameter.query.filter_by(stream_parameter_name=id).first_or_404()
     return jsonify(parameter.to_json())
-
-@api.route('/organization', methods=['GET'])
-def get_organizations():
-    organizations = [o.serialize() for o in Organization.query.all()]
-    return jsonify(organizations=organizations)
-
-@api.route('/get_data')
```
Comment (birdage) [inline ooiservices/app/main/routes.py]: @FBRTMaka this had changed in the most recent version?

**Comment by birdage** [comment]
@FBRTMaka your changes will kill changes i made yesterday to handle uframe data.....

**Comment by FBRTMaka** [comment]
I do have that in this feature branch

**Comment by FBRTMaka** [comment]
it won’t kill it, just move it to a proper home

**Comment by FBRTMaka** [comment]
I’ll make sure it all gets in there

**Comment by birdage** [comment]
@FBRTMaka cheers.

**Comment by birdage** [comment]
@FBRTMaka this closed for a reason? not merged?

**Pull request closed without merging**